src/Standards/Generic/Tests/PHP/LowerCaseKeywordUnitTest.inc.fixed

```diff
@@ -16,4 +16,6 @@ goto a;
 function (array $a) {}
 const PRIVATE;
 HttpStatus::CONTINUE;
-function
```

Comment:
This removes a very specific test case that is checking for an error that used to occur when the file ends with `function` (as happens when using check-as-you-type plugins). The commit is here: https://github.com/squizlabs/PHP_CodeSniffer/commit/88d3f29ffab05f68544529b63536d3ab82b7cb7f

When you see syntax errors in tests files, they are almost always there for a specific reason, so it's best to check the test file history just to be sure it's not a mistake.